libimage/pull.go

```diff
@@ -8,15 +8,13 @@ import (
 
 	"github.com/containers/common/pkg/config"
 	dirTransport "github.com/containers/image/v5/directory"
-	dockerRegistryTransport "github.com/containers/image/v5/docker"
```

Comment:
Can we leave this as registryTransport ...

I think that makes it cleaner and easier to understand for future users of this library.  Especially since there is a docker-daemon transport.  Also eliminates lots of "docker" adjectives from the code.

Reply:
Sure.  I will do the rename then in all files so all imports are consistent.

Reply:
Done.